internal/database/schema.go

```diff
@@ -94,11 +87,13 @@ var schema = []sqlTable{
 	createSQLTable("sources_datasources").
 		VarCharNonNullColumn("name").
 		VarCharNonNullColumn("source_name").
+		VarCharColumn("secret_name").
```

Comment:
MySQL won't let us use `TextColumn` with `ForeignKey`, because it builds an index for those and cannot put an index on TEXT without a "prefix count", which basically is the `VarChar(255)` we'll put there for `VarCharColumn`. Postgres and SQLite didn't care.